app/test_main.py

```diff
@@ -1,3 +1,23 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_years,dog_years,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
```

Comment:
You should additionally test cases when age is negative, cat and dog ages differ, and when you receive inappropriate data types for years to test an exception raised according to the checklist. You could make another function for testing exceptions.